deploy/olm-catalog/operand-deployment-lifecycle-manager/1.1.0/operand-deployment-lifecycle-manager.v1.1.0.clusterserviceversion.yaml

```diff
@@ -581,19 +581,19 @@ spec:
     operator/operand you want to install in the cluster\n\n**NOTE:** The ODLM-managed
     operator subscriptions have a label **\"operator.ibm.com/opreq-control\": \"true\"**.
     We use this label to distinguish if an operator is managed by ODLM or not.\n\n##
-    OperandRegistry\n\nOperandRegistry defines the individual service state. For example,
-    an individual service should be present or absent.\n\n**NOTE:** When the ODLM
-    operator is deployed, it generates a default OperandRegistry instance. You can
-    edit the instance as required. \n\nFollowing is an example of the OperandRegistry
-    CR: \n\n **NOTE:**The \"name\" parameter must be unique for each entry.\n\n```yaml\napiVersion:
-    odlm.ibm.com/v1alpha1\nkind: OperandRegistry\nmetadata:\n  name: common-service
-    [1]\n  namespace: ibm-common-services [2]\nspec:\n  operators:\n  - name: ibm-metering-operator
-    [3]\n    namespace: ibm-common-services [4]\n    channel: alpha [5]\n    packageName:
-    ibm-metering-operator-app [6]\n    sourceName: opencloud-operators [7]\n    sourceNamespace:
-    openshift-marketplace [8]\n    description: The service used to meter workloads
-    in a kubernetes cluster [9]\n```\n\nThe Operand (Deployment) Registry Custom Resource
-    (CR) lists OLM Operator information for operands that may be requested for installation
-    and/or access by an application that runs in a namespace.  The registry CR specifies:\n\n1.
+    OperandRegistry\n\nOperandRegistry defines the OLM information, like channel and
+    catalog source, for each operator. \n\n**NOTE:** When the ODLM operator is deployed,
```

Comment:
1.1.0 has been released, after this PR, does the release also need to update?

Reply:
@gyliu513 Yes, I will update the CSV description into the branch `release-1.1` as well and publish ODLM with the updated description to the RH.